packages/rabbitmq/src/rabbitmq.interfaces.ts

```diff
@@ -176,10 +176,23 @@ export type RabbitMQHandlers = Record<
   string,
   MessageHandlerOptions | MessageHandlerOptions[]
 >;
+export type RabbitMQUriConfig = RabbitMQUriConfigObject | string;
+
+export interface RabbitMQUriConfigObject {
```

Comment:
@DanielShvarts2004 Does the amq lib offers the config object types? If so i'd prefer to simply import those